CoreFoundation/Preferences.subproj/CFXMLPreferencesDomain.c

```diff
@@ -198,7 +198,7 @@ static void _loadXMLDomainIfStale(CFURLRef url, _CFXMLPreferencesDomain *domain)
     domain->_lastReadTime = CFAbsoluteTimeGetCurrent();
 }
 
-static CFTypeRef fetchXMLValue(CFTypeRef context, void *xmlDomain, CFStringRef key) {
+static CFTypeRef fetchXMLCopyValue(CFTypeRef context, void *xmlDomain, CFStringRef key) {
```

Comment:
I think `copyXMLValue` would be a more standard name for this.

Reply:
fixed